pkgs/development/python-modules/eventlet/default.nix

```diff
@@ -43,16 +43,17 @@ buildPythonPackage rec {
     six
   ];
 
-  nativeCheckInputs = [ pytestCheckHook ];
+  nativeCheckInputs = [
+    libredirect.hook
+    pytestCheckHook
+  ];
 
-  # libredirect is not available on darwin
   # tests hang on pypy indefinitely
-  doCheck = !stdenv.hostPlatform.isDarwin && !isPyPy;
+  doCheck = !isPyPy;
```

Comment:
Have you tested this on darwin?  It's broken now on Hydra:
https://hydra.nixos.org/build/291685189
https://hydra.nixos.org/build/291685193#tabs-buildsteps

Reply:
It's in `staging-next` already; /cc the PR https://github.com/NixOS/nixpkgs/pull/386822

Reply:
I don't know.  When multiple issues happen, some may hide some others 🤷🏽

Reply:
#387665